### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by fpsapc**
Hello Reviewer,
- Remove old app data (text, images, styles)
- Create a components directory
- Inside components create a Calculator.js file
- Create calculator UI implement (JSX and CSS)
- Call Calculator from App.js also add some styles in App.css

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          <button type="button" className="singleButton">3</button>
+          <button type="button" className="singleButton">+</button>
+          <button type="button" className="doubleButton">0</button>
+          <button type="button" className="singleButton">.</button>
+          <button type="button" className="singleButton">=</button>
+        </div>
+      </div>
+    </>
+  );
+}
```
Comment (thecodechaser) [inline src/components/Calculator.js]: - [x] Please make sure to create at least one child component for the Calculator component as requested in the project requirements. (I would suggest separating the buttons code in a different component and importing it here in the Calculator component).

image

File: src/App.css [hunk 2]
```diff
@@ -1,39 +1,46 @@
...
-  from {
-    transform: rotate(0deg);
-  }
+.keypad :nth-child(4),
+.keypad :nth-child(8),
+.keypad :nth-child(12),
+.keypad :nth-child(16),
+.keypad :nth-child(19) {
+  background-color: #ffb05c;
+}
```
Comment (thecodechaser) [inline src/App.css]: - [x] Please make sure to use the correct color for the operators' buttons background `#F5913E` to follow the wireframe design.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Amazinggracee👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️

- [x] Please make sure to delete the `logo.svg` file in the src directory as requested in the project requirements to delete all the boilerplate from the app.
- [x] Please make sure to delete the styles in the `index.css` file in the src directory as requested in the project requirements to delete all the boilerplate from the app.
- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @fpsapc,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️
- Readme file professional ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**